README.rst

```diff
@@ -1,43 +1,111 @@
-===============================
-DateParser
-===============================
+====================================================
+dateparser -- python parser for human readable dates
+====================================================
 
-.. image:: https://badge.fury.io/py/dateparser.png
-    :target: http://badge.fury.io/py/dateparser
+.. image:: https://img.shields.io/travis/scrapinghub/dateparser/master.svg?style=flat-square
+    :target: https://travis-ci.org/scrapinghub/dateparser
+    :alt: travis build status
 
-.. image:: https://travis-ci.org/scrapinghub/dateparser.png?branch=master
-        :target: https://travis-ci.org/scrapinghub/dateparser
+.. image:: https://img.shields.io/pypi/dd/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser/
+    :alt: pypi downloads per day
 
-.. image:: https://pypip.in/d/dateparser/badge.png
-        :target: https://pypi.python.org/pypi/dateparser
+.. image:: https://img.shields.io/pypi/v/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser
+    :alt: pypi version
 
 
-Date parsing library designed to make it easy parsing dates commonly found in web pages
+`dateparser` provides modules to easily parse localized dates in almost
+any string formats commonly found on web pages.
 
 
-* Free software: BSD license
-* Documentation: https://dateparser.readthedocs.org.
-
 Features
 --------
+* Generic parsing of dates in English, Spanish, Dutch, Russian and several other langauges and formats.
+* Generic parsing of contextual dates like: ``'1 min ago'``, ``'2 weeks ago'``, ``'3 months, 1 week and 1 day ago'``.
+* Generic parsing of dates with time zones abbreviations like: ``'August 14, 2015 EST'``, ``'July 4, 2013 PST'``.
+* Extensive test coverage.
+
+
+Usage
+-----
+The most straightforward way is to use the :func:`dateparser.parse` function,
+that wraps around most of the functionality in the module.
+
+.. automodule:: dateparser.__init__
```

Comment:
Do we need `__init__` here? It appears as `dateparser.__init__.parse` in compiled docs.